campaigns/utils.py

```diff
@@ -0,0 +1,13 @@
+from datetime import date
+
+from sos_brazil.exceptions import BadEndDateException, BadStartDateException
+
+
+def check_dates(start_date, end_date):
+    if start_date < date.today():
+        raise BadStartDateException()
+
+    elif end_date <= start_date:
```

Comment:
I believe, by our discussions, that the user can create an `end_date` equals to `start_date`, like an "one day campaign" thing

Reply:
fix: user can create and change end_date equals start_date